scripts/mktcli.js

```diff
@@ -290,7 +304,7 @@ const main = {
           const asset = await getAssetFromCLIArgs(assetId, assetType)
 
           const events = await BlockchainEvent.findByAnyArgs(
-            ['assetId', '_landId'],
+            ['assetId', 'landId', '_landId', '_estateId'],
```

Comment:
Maybe we could add also `_tokenId`

Reply:
Hmm What do you think if we create a const like

`const argsId = ['assetId', 'landId', '_landId', '_estateId', '_tokenId']`

IDK a good name: `argsId`, `wordToMatchId`, `matchId`, `argsNames`

Reply:
Agree with abstracting this!